examples/finite_prime_field.rs

```diff
@@ -0,0 +1,254 @@
+use noether::{
+    AssociativeAddition, AssociativeMultiplication, CommutativeAddition, CommutativeMultiplication,
+    Distributive, FiniteField,
+};
+use num_traits::{Euclid, Inv, One, Zero};
```

Comment:
Inv/One/Zero is having an inverse/multiplicative identity (one)/additive identity (zero)? Note that in rings and higher you have additive vs multiplicative inverses. I assume Euclid is Euclidean division based on the below.

Reply:
Yep, along with the neg trait.

Reply:
Added comments explaining this in the library.